pymatviz/ptable.py

```diff
@@ -522,16 +619,71 @@ def add_colorbar(
         )
 
         # Set colorbar title
-        title_kwargs = title_kwargs or {}
-        title_kwargs.setdefault("fontsize", 12)
-        title_kwargs.setdefault("pad", 10)
-        title_kwargs["label"] = title
-
         cbar_ax.set_title(**title_kwargs)
 
+    def add_elem_type_legend(
```

Comment:
i think something got lost here. the colors no longer default to `ELEM_CLASS_COLORS`?

Reply:
It still is, I packed everything inside the following property:
```python
    @elem_type_colors.setter
    def elem_type_colors(self, elem_type_colors: dict[str, str] | None) -> None:
        default_elem_type_colors: dict[str, str] = {
            "Diatomic Nonmetal": "green",
            "Noble Gas": "purple",
            "Alkali Metal": "red",
            "Alkaline Earth Metal": "orange",
            "Metalloid": "darkgreen",
            "Polyatomic Nonmetal": "teal",
            "Transition Metal": "blue",
            "Post Transition Metal": "cyan",
            "Lanthanide": "brown",
            "Actinide": "gray",
            "Nonmetal": "green",
            "Halogen": "teal",
            "Metal": "lightblue",
            "Alkaline Metal": "magenta",
            "Transactinide": "olive",
        }

        self._elem_type_colors = default_elem_type_colors | (elem_type_colors or {})
```